dev/inc/SharedHelpers.h

```diff
@@ -10,6 +10,7 @@ class SharedHelpers
 public:
     static bool IsAnimationsEnabled();
     static winrt::IInspectable FindResourceOrNull(const std::wstring_view& resource, const winrt::ResourceDictionary& resources);
+    static winrt::IInspectable FindResource(const std::wstring_view& resource, const winrt::ResourceDictionary& resources, const winrt::IInspectable& defaultValue);
     static bool IsInDesignMode();
```

Comment:
How about a default value of nullptr for the defaultValue parameter so that FindResourceOrNull can be removed? FindResource gives a default value of nullptr by default then.

Reply:
Done.